SetupDataPkg/ConfApp/SetupConf.c

```diff
@@ -274,10 +273,11 @@ ApplySettings (
   UINTN       Lsv           = 0;
   BOOLEAN     ResetRequired = FALSE;
 
-  UINTN       b64Size;
-  UINTN       ValueSize;
-  UINT8       *ByteArray = NULL;
-  CONST VOID  *SetValue  = NULL;
+  UINTN                          b64Size;
+  UINTN                          ValueSize;
+  UINT8                          *ByteArray = NULL;

```

Comment:
Since this file is being modified, please move the initializations out of the function declarations. 
```suggestion
  UINT8                          *ByteArray;

```

Add ByteArray = NULL; after all the declarations are done.

Reply:
Both ByteArray and SetValue initialized after all declarations are done